BackendApartmentReservation/BackendApartmentReservation/Files/FilesController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using BackendApartmentReservation.Authentication.AuthorizationRequirements.OrganizerOnly;
+using BackendApartmentReservation.Infrastructure.Authorization;
+
+namespace BackendApartmentReservation.Employees
+{
+    using System.Threading.Tasks;
+    using Database.Entities;
+    using Interfaces;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.AspNetCore.Http;
+    using System.IO;
+    using Microsoft.AspNetCore.Authorization;
+    
+    [ApiController]
+    public class FilesController : AuthorizedController
+    {
+        private readonly IFileManager _fileManager;
+
+
+        public FilesController(IFileManager fileManager)
+        {
+            _fileManager = fileManager;
+        }
+
+        [HttpPost]
+        [OrganizerOnly]
+        [Route("files")]
+        public async Task<IActionResult> Create(IFormFile file)
+        {
+            long size = file.Length;
```

Comment:
Do not use `db` entities in the controller. Manager should do all the work starting from stream.